evap/staff/importers/json.py

```diff
@@ -284,6 +284,22 @@ def _import_course_programs(self, course: Course, data: ImportEvent) -> None:
         ]
         course.programs.set(programs)
 
+    def _import_course_from_unused_exam(self, data: ImportEvent) -> Course | None:
+        splitted_title = data["title"].split(":", 1)
+        if len(splitted_title) < 2:
+            return None
+        prefix = splitted_title[0].strip()
+
+        try:
+            course_type = CourseType.objects.get(import_names__contains=[prefix])
```

Comment:
this should be `course_type = CourseType.objects.get(import_names__in=[prefix])` to ensure exact matches

Reply:
looks like array fields are weird - you can ignore this :)